sycl/include/sycl/ext/oneapi/matrix/matrix-unified-utils.hpp

```diff
@@ -0,0 +1,26 @@
+//===------- matrix-unified.hpp - SYCL matrix extension ----*- C++ -*------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// ===--------------------------------------------------------------------=== //
+
+#pragma once
+namespace sycl {
+__SYCL_INLINE_VER_NAMESPACE(_V1) {
+namespace ext {
+namespace oneapi {
+namespace experimental {
+namespace matrix {
+
+enum class use { a, b, accumulator };
+
+enum class layout { row_major = 0, col_major = 1, dynamic = 3 };
```

Comment:
what was the reason behind this change:
enum class layout { row_major, col_major, dynamic };
to 
enum class layout { row_major = 0, col_major = 1, dynamic = 3 };
?

Reply:
because layout::packed carry the value 2